astro/src/content/docs/apis/_user-response-body-base.mdx

```diff
@@ -0,0 +1,328 @@
+  <APIField name={props.base_field_name + '.expiry'} type="Long">
+  The expiration [instant](/docs/reference/data-types#instants) of the User's account. An expired user is not permitted to login.
+</APIField>
+  <APIField name={props.base_field_name + '.firstName'} type="String">
+  The first name of the User.
+</APIField>
+  <APIField name={props.base_field_name + '.fullName'} type="String">
+  The User's full name as a separate field that is not calculated from <InlineField>firstName</InlineField> and <InlineField>lastName</InlineField>.
+</APIField>
+  <APIField name={props.base_field_name + '.id'} type="UUID">
+  The User's unique Id.
+</APIField>
+  <APIField name={props.base_field_name + '.identities'} type="Array" since="1.99.9">
+  The list of identities that exist for a User.
+</APIField>
+  <APIField name={props.base_field_name + '.identities[x].displayValue'} type="String" since="1.99.9">
+  The display value for the identity. Only used for `username` type identities. If the unique username feature is not enabled, this value
+  will be the same as <InlineField>{props.base_field_name}.identities``[x]``.value</InlineField>. Otherwise, it will be the username the User has chosen. For
+  primary username identities, this will be the same value as <InlineField>{props.base_field_name}.username</InlineField>.
+</APIField>
+  <APIField name={props.base_field_name + '.identities[x].insertInstant'} type="Long" since="1.99.9">
+  The [instant](/docs/reference/data-types#instants) when the identity was created.
+</APIField>
+  <APIField name={props.base_field_name + '.identities[x].lastLoginInstant'} type="Long" since="1.99.9">
+  The [instant](/docs/reference/data-types#instants) when the identity was last used to login. If a User has multiple identity types (`username`, `email`, and `phoneNumber`), then this value will represent the specific identity they last used to login. This contrasts with <InlineField>{props.base_field_name}.lastLoginInstant</InlineField>, which represents the last time any of the User's identities was used to login.
+
+  {/* more */}
+  <Aside type="note">
+    Identity provider logins will not be reflected here (see <InlineField>{props.base_field_name}.lastLoginInstant</InlineField> for a field that includes identity provider logins).
+  </Aside>
+</APIField>
+  <APIField name={props.base_field_name + '.identities[x].lastUpdateInstant'} type="Long" since="1.99.9">
+  The [instant](/docs/reference/data-types#instants) when the identity was last updated.
+</APIField>
+  <APIField name={props.base_field_name + '.identities[x].moderationStatus'} type="String" since="1.99.9">
+  <ModerationStatusResponse />
+  Only used for `username` type identities. For primary username identities, this will be the same value as <InlineField>{props.base_field_name}.usernameStatus</InlineField>.
+</APIField>
+  <APIField name={props.base_field_name + '.identities[x].primary'} type="Boolean" since="1.99.9">
+  Reserved for future use.
```

Comment:
How much do we want to talk about this now?

Reply:
My vote is none. I think we should remove the field from API responses in `app`.

Reply:
removed. and capture follow up work to hide field